src/displayapp/screens/Steps.cpp

```diff
@@ -18,9 +18,10 @@ Steps::Steps(Pinetime::Applications::DisplayApp* app,
   stepsArc = lv_arc_create(lv_scr_act(), nullptr);
 
   lv_obj_set_style_local_bg_opa(stepsArc, LV_ARC_PART_BG, LV_STATE_DEFAULT, LV_OPA_0);
+  lv_obj_set_style_local_line_color(stepsArc, LV_ARC_PART_BG, LV_STATE_DEFAULT, Colors::bgAlt);
   lv_obj_set_style_local_border_width(stepsArc, LV_ARC_PART_BG, LV_STATE_DEFAULT, 2);
   lv_obj_set_style_local_radius(stepsArc, LV_ARC_PART_BG, LV_STATE_DEFAULT, 0);
-  lv_obj_set_style_local_line_color(stepsArc, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, LV_COLOR_BLUE);
+  lv_obj_set_style_local_line_color(stepsArc, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, LV_COLOR_MAKE(0, 0x40, 0xff));
```

Comment:
Giving the color a name would be nice, as I'm very bad at translating hex to actual colors in my head ;)

should we define the color as "lightBlue" in `InfiniTimeTheme.h` to reuse somewhere else?

Reply:
Yes, I should've mentioned, I was thinking of getting #1506 merged first and then use the same color here.